### PR TITLE
Added CVE-2020-10717

### PR DESCRIPTION
**Pull request by tausif-rh**

### CODE REVIEW

File: 2020/10xxx/CVE-2020-10717.json [hunk 1]
```diff
@@ -4,15 +4,68 @@
...
+            }
+        ]
+    },
+    "references": {
+        "reference_data": [
+            {
+                "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-10717",
+                "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-10717",
+                "refsource": "CONFIRM"
+            }
```
Comment (iamleot) [inline 2020/10xxx/CVE-2020-10717.json:50]: Please add corresponding patches:

 https://lists.gnu.org/archive/html/qemu-devel/2020-05/msg00143.html
 https://lists.gnu.org/archive/html/qemu-devel/2020-05/msg00141.html

...and discussion on oss-security@ ML:

 https://lists.gnu.org/archive/html/qemu-devel/2020-05/msg00141.html

**Comment by iamleot** [comment]
@cve-team Can we please wait for possible feedbacks/applying changes suggested?

File: 2020/10xxx/CVE-2020-10717.json (hunk 1 above)
Comment (tausif-rh) [reply 2020/10xxx/CVE-2020-10717.json:50]: Added the reference data at https://github.com/CVEProject/cvelist/pull/3768

Comment (iamleot) [reply 2020/10xxx/CVE-2020-10717.json:50]: Thanks @tausif-rh!

**Comment by tausif-rh** [comment]
> Hello @tausif-rh,
> please add upstream patches and discussion on oss-security@ ML.
> 
> In general, it would be better if you always do that instead of only having a layer of indirection via the Red Hat Bugzilla. Can you please do that for future CVEs?
> 
> Thanks!

Hi @iamleot 

We currently don't have a feature that parses all the URLs from the bugzilla comments. We're looking for ways to improve this. Thank you.

**Pull request merged**